client/pkg/git/git.go

```diff
@@ -106,5 +106,6 @@ func findRemote(host string) (string, error) {
 
 // RemoteURL returns the git URL of app.
 func RemoteURL(host, appID string) string {
+	host = strings.Split(host, ":")[0]
```

Comment:
would you mind adding a comment here say that this is intended to strip out the port number?